main.go

```diff
@@ -55,7 +55,7 @@ func main() {
 
 	files, err := testRunner.GetFiles()
 	if err != nil {
-		logErrorAndExit(16, "Couldn't get files: %v", err)
+		logErrorAndExit(16, "couldn't get files: %v", err)
```

Comment:
I think this is fine with a Capital letter. The conventions of not using capital letter is for error _message_, rather than the _log_.